### PR TITLE
Resco Reports connector artifacts

### PR DESCRIPTION
**Pull request by JuroJ**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by JuroJ** [comment]
@microsoft-github-policy-service agree [company="Resco"]

**Comment by JuroJ** [comment]
@microsoft-github-policy-service agree company="Resco"

**Comment by Amjed-Ayoub** [comment]
Hello @JuroJ,

Thank you so much for submitting your new connector artifacts.

As we ask all Verified Partner to register their connectors, may I ask you please to register your connector by filling out our [registration form](https://aka.ms/ConnectorRegistration).

Please let me know if you have any questions.

Thank you so much again.

**Comment by JuroJ** [comment]
Done

**Comment by JuroJ** [comment]
I've submitted the form once more (actually the 3rd time). Here's the proof: 
image

**Comment by Amjed-Ayoub** [comment]
> I've submitted the form once more (actually the 3rd time). Here's the proof: image

@JuroJ Thank you so much for the confirmation. 
Actually, you brought our attention to very critical issue. We are not receiving any registration form! 
We will move forward with you submission and review your PR soon.

Thank you so much again for your patience and understanding.

File: certified-connectors/RescoReports/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Resco Reports
```
Comment (Amjed-Ayoub) [inline certified-connectors/RescoReports/README.md:1]: @JuroJ Please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

File: certified-connectors/RescoReports/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,400 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Resco Reports",
+    "description": "Resco Reports provides you with a highly customizable tool for generating reports and documents based on your Dataverse / Dynamics data. This connector makes it possible to use the Resco Mobile Report templates to automate document generation or any other back-office scenarios.",
+    "contact": {
+      "name": "Resco experts are ready to listen and help",
+      "url": "https://www.resco.net/contact-support/"
```
Comment (Amjed-Ayoub) [inline certified-connectors/RescoReports/apiDefinition.swagger.json:9]: @JuroJ Please add an "email" property in the contact section. Your support contact email.

File: certified-connectors/RescoReports/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+            "$ref": "#/responses/error"
+          }
+        }
+      }
+    },
+    "/resco_mobilereports": {
+      "get": {
+        "summary": "List reports",
+        "description": "Get list of records",
+        "operationId": "Report-List",
```
Comment (Amjed-Ayoub) [inline certified-connectors/RescoReports/apiDefinition.swagger.json:188]: @JuroJ all operationIds should be in Camel Case. Please chagne to "ReportList".
Please fix all operationIds in the swagger file.

File: certified-connectors/RescoReports/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
```
Comment (Amjed-Ayoub) [inline certified-connectors/RescoReports/apiProperties.json:7]: @JuroJ can you please confirm if your connector is using Azure Active Directory as an identity provider?

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/RescoReports/README.md (hunk 1 above)
Comment (JuroJ) [reply certified-connectors/RescoReports/README.md]: Done

File: certified-connectors/RescoReports/apiDefinition.swagger.json (hunk 2 above)
Comment (JuroJ) [reply certified-connectors/RescoReports/apiDefinition.swagger.json]: Done

File: certified-connectors/RescoReports/apiDefinition.swagger.json (hunk 3 above)
Comment (JuroJ) [reply certified-connectors/RescoReports/apiDefinition.swagger.json]: Done

File: certified-connectors/RescoReports/apiDefinition.swagger.json [hunk 5]
```diff
@@ -185,7 +186,7 @@
       "get": {
         "summary": "List reports",
         "description": "Get list of records",
-        "operationId": "Report-List",
+        "operationId": "reportList",
```
Comment (Amjed-Ayoub) [inline certified-connectors/RescoReports/apiDefinition.swagger.json:189]: @JuroJ all operationIds should be in Camel Case. Please change to "ReportList". It should start with a capital letter. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operationIds in the swagger file.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Review by Amjed-Ayoub — APPROVED** [review]

**Pull request merged**